manifests/r/Rainmeter/Rainmeter/Beta/4.4.0.3507/Rainmeter.Rainmeter.Beta.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter.Beta
+PackageVersion: 4.4.0.3507
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/rainmeter/rainmeter/releases/download/v4.4.0.3507/Rainmeter-4.4-r3507-beta.exe
+  InstallerSha256: BC248921FB2DA6458F220965B9EB7C9933D32EE80E00A7BFD17C033B3B995163
+  InstallerSwitches:
+    Silent: /S
+#    SilentWithProgress: 
```

Comment:
```suggestion
```
This part can be removed because winget knows nullsoft has /S for silent. Thanks for adding silent installation switch. 😁